core/templates/pages/exploration-editor-page/translation-tab/audio-translation-bar/audio-translation-bar.directive.ts

```diff
@@ -409,12 +402,13 @@ angular.module('oppia').directive('audioTranslationBar', [
             var audioTranslation = getAvailableAudio(
               $scope.contentId, $scope.languageCode);
             if (audioTranslation) {
-              $q.when(AudioPlayerService.load(audioTranslation.filename))
+              (AudioPlayerService.loadAsync(audioTranslation.filename))
```

Comment:
```suggestion
              AudioPlayerService.loadAsync(audioTranslation.filename).then(
                function() {
```
optional* (To avoid unwanted parenthesis)

Reply:
Done!